msgid ""
msgstr ""
"Project-Id-Version: ubuntu-translations\n"
"Report-Msgid-Bugs-To: FULL NAME <EMAIL@ADDRESS>\n"
"POT-Creation-Date: 2023-06-09 17:35+0000\n"
"PO-Revision-Date: 2024-03-07 08:58\n"
"Last-Translator: meequz <meequz@gmail.com>\n"
"Language-Team: Esperanto\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2024-03-07 07:56+0000\n"
"X-Generator: Launchpad (build 2e4955e3d556e61e2eb54acf7a827d17e9e46822)\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Crowdin-Project: ubuntu-translations\n"
"X-Crowdin-Project-ID: 485161\n"
"X-Crowdin-Language: eo\n"
"X-Crowdin-File: /main/noble/en_GB/po-usb-creator_usbcreator-be.po\n"
"X-Crowdin-File-ID: 57\n"
"Language: eo_UY\n"

#: .././gui/usbcreator-gtk.ui.h:1
msgid "Installation Failed"
msgstr ""

#: .././gui/usbcreator-gtk.ui.h:2
msgid "The installation failed.  Please see ~/.cache/usb-creator.log for more details.\n"
msgstr ""

#: .././gui/usbcreator-gtk.ui.h:4
msgid "Installation Complete"
msgstr ""

#: .././gui/usbcreator-gtk.ui.h:5
msgid "Test Disk"
msgstr ""

#: .././gui/usbcreator-gtk.ui.h:6
msgid "Installation is complete.  You may now run Ubuntu on other computers by booting them with this drive inserted."
msgstr ""

#. #-#-#-#-#  usbcreator.pot (PACKAGE VERSION)  #-#-#-#-#
#. set title of progress window (same as gtk frontend)
#: .././gui/usbcreator-gtk.ui.h:7 .././usbcreator/frontends/kde/frontend.py:429
#: usbcreator/frontends/kde/frontend.py:429
msgid "Installing"
msgstr ""

#: .././gui/usbcreator-gtk.ui.h:8 .././usbcreator/frontends/kde/frontend.py:505
#: usbcreator/frontends/kde/frontend.py:505
msgid "Quit the installation?"
msgstr ""

#: .././gui/usbcreator-gtk.ui.h:9 .././usbcreator/frontends/kde/frontend.py:506
#: usbcreator/frontends/kde/frontend.py:506
msgid "Do you really want to quit the installation now?"
msgstr ""

#: .././gui/usbcreator-gtk.ui.h:10
msgid "Make Startup Disk"
msgstr ""

#: .././gui/usbcreator-gtk.ui.h:11
msgid "To try or install Ubuntu from a removable disk, it needs to be set up as a startup disk."
msgstr ""

#: .././gui/usbcreator-gtk.ui.h:12
msgid "Source disc image (.iso):"
msgstr ""

#: .././gui/usbcreator-gtk.ui.h:13
msgid "Other..."
msgstr ""

#: .././gui/usbcreator-gtk.ui.h:14
msgid "Disk to use:"
msgstr ""

#: .././gui/usbcreator-gtk.ui.h:15
msgid "Erase Disk"
msgstr ""

#: .././desktop/usb-creator-kde.desktop.in.h:1
#: .././desktop/usb-creator-gtk.desktop.in.h:1
msgid "Startup Disk Creator"
msgstr ""

#: .././desktop/usb-creator-kde.desktop.in.h:2
#: .././desktop/usb-creator-gtk.desktop.in.h:2
msgid "Create a startup disk using a CD or disc image"
msgstr ""

#: .././dbus/com.ubuntu.usbcreator.policy.in.h:1
msgid "Image the device"
msgstr ""

#: .././dbus/com.ubuntu.usbcreator.policy.in.h:2
msgid "System policy prevents writing a disk image to this device"
msgstr ""

#: .././dbus/com.ubuntu.usbcreator.policy.in.h:3
msgid "Run KVM"
msgstr ""

#: .././dbus/com.ubuntu.usbcreator.policy.in.h:4
msgid "System policy prevents running KVM"
msgstr ""

#: .././usbcreator/frontends/kde/frontend.py:200
#: .././usbcreator/frontends/gtk/frontend.py:655
#: usbcreator/frontends/kde/frontend.py:200
msgid "Installation failed."
msgstr ""

#: .././usbcreator/frontends/kde/frontend.py:355
#: .././usbcreator/frontends/gtk/frontend.py:430
#: usbcreator/frontends/kde/frontend.py:355
msgid "The device is not large enough to hold this image."
msgstr ""

#. #-#-#-#-#  usbcreator.pot (PACKAGE VERSION)  #-#-#-#-#
#. This here filter is for kfiledialog, no clue if it will ever make a return
#. filter = '*.iso|' + _('CD Images') + '\n*.img|' + _('Disk Images')
#: .././usbcreator/frontends/kde/frontend.py:387
#: .././usbcreator/frontends/gtk/frontend.py:548
#: usbcreator/frontends/kde/frontend.py:387
msgid "CD Images"
msgstr ""

#: .././usbcreator/frontends/kde/frontend.py:387
#: .././usbcreator/frontends/gtk/frontend.py:549
#: usbcreator/frontends/kde/frontend.py:387
msgid "Disk Images"
msgstr ""

#: .././usbcreator/frontends/kde/frontend.py:417
#: .././usbcreator/frontends/gtk/frontend.py:577
#: usbcreator/frontends/kde/frontend.py:417
msgid "Are you sure you want to write the disc image to the device?"
msgstr ""

#: .././usbcreator/frontends/kde/frontend.py:418
#: .././usbcreator/frontends/gtk/frontend.py:578
#: usbcreator/frontends/kde/frontend.py:418
msgid "All existing data will be lost."
msgstr ""

#: .././usbcreator/frontends/kde/frontend.py:427
#: usbcreator/frontends/kde/frontend.py:427
msgid "Cancel"
msgstr ""

#: .././usbcreator/frontends/kde/frontend.py:438
#: usbcreator/frontends/kde/frontend.py:438
msgid "Starting up"
msgstr ""

#: .././usbcreator/frontends/kde/frontend.py:447
#: usbcreator/frontends/kde/frontend.py:447
msgid "You must select both source image and target device first."
msgstr ""

#: .././usbcreator/frontends/kde/frontend.py:458
#: .././usbcreator/frontends/gtk/frontend.py:611
#: usbcreator/frontends/kde/frontend.py:458
msgid "Finishing..."
msgstr ""

#: .././usbcreator/frontends/kde/frontend.py:477
#: usbcreator/frontends/kde/frontend.py:477
msgid "The installation is complete.  You may now reboot your computer with this device inserted to try or install Ubuntu."
msgstr ""

#: .././usbcreator/frontends/kde/frontend.py:489
#: usbcreator/frontends/kde/frontend.py:489
msgid "Retry?"
msgstr ""

#: .././usbcreator/frontends/gtk/frontend.py:342
msgid "CD-Drive/Image"
msgstr ""

#: .././usbcreator/frontends/gtk/frontend.py:355
msgid "OS Version"
msgstr ""

#: .././usbcreator/frontends/gtk/frontend.py:365
msgid "Size"
msgstr ""

#: .././usbcreator/frontends/gtk/frontend.py:500
msgid "Device"
msgstr ""

#: .././usbcreator/frontends/gtk/frontend.py:516
msgid "Label"
msgstr ""

#: .././usbcreator/frontends/gtk/frontend.py:526
msgid "Capacity"
msgstr ""

#: .././usbcreator/frontends/gtk/frontend.py:584
msgid "Starting up..."
msgstr ""

#: .././usbcreator/frontends/gtk/frontend.py:609
#, python-format
msgid "%d%% complete"
msgstr ""

#: .././usbcreator/install.py:92
#, python-format
msgid "The extension \"%s\" is not supported."
msgstr ""

#: .././usbcreator/install.py:101
#, python-format
msgid "An uncaught exception was raised:\n"
"%s"
msgstr ""

#: .././usbcreator/install.py:105
msgid "Writing disk image..."
msgstr ""

#: .././usbcreator/install.py:106
#, python-format
msgid "Could not write the disk image (%(source)s) to the device (%(device)s)."
msgstr ""

#: .././bin/usb-creator-kde:64
msgid "provide a source image to pre-populate the UI."
msgstr ""

#: .././bin/usb-creator-kde:69
msgid "allow writing to system-internal devices"
msgstr ""

